.github/workflows/ci-core-reusable.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           ci_run zk contract build
 
       - name: Contracts unit tests
-        run: ci_run zk test l1-contracts
+        run: ci_run yarn l1-contracts test
```

Comment:
let's then remove the 'l1-contracts' from zk test..

(infrastructure/tests.ts)